<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-HHURL3J" name="sikuli_classfiles.Test1" tests="1" failures="0" timestamp="13 May 2020 18:02:54 GMT" time="26.270" errors="1">
  <testcase name="fileUpload" time="26.270" classname="sikuli_classfiles.Test1">
    <error type="org.sikuli.script.FindFailed" message="can not find C:\Intellipat_March 22\choosefile.PNG in S(0)[0,0 1536x864]">
      <![CDATA[FindFailed: can not find C:\Intellipat_March 22\choosefile.PNG in S(0)[0,0 1536x864]
  Line 2189, in file Region.java

at org.sikuli.script.Region.handleFindFailedShowDialog(Region.java:2189)
at org.sikuli.script.Region.handleFindFailed(Region.java:2134)
at org.sikuli.script.Region.wait(Region.java:2546)
at org.sikuli.script.Region.find(Region.java:2205)
at org.sikuli.script.Region.getLocationFromTarget(Region.java:3067)
at org.sikuli.script.Region.click(Region.java:3534)
at org.sikuli.script.Region.click(Region.java:3519)
at sikuli_classfiles.Test1.fileUpload(Test1.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
    </error>
  </testcase> <!-- fileUpload -->
</testsuite> <!-- sikuli_classfiles.Test1 -->
